lib/src/components/NxCloseButton/NxCloseButton.tsx

```diff
@@ -16,7 +16,12 @@ const NxCloseButton = forwardRef<HTMLButtonElement, ButtonHTMLAttributes<HTMLBut
       const btnClasses = classnames('nx-btn--close', className);
 
       return (
-        <NxButton ref={ref} type="button" className={btnClasses} variant="icon-only" { ...otherProps }>
+        <NxButton ref={ref}
+                  type="button"
+                  className={btnClasses}
+                  variant="icon-only"
+                  { ...otherProps }
+                  aria-label="Dismiss">
```

Comment:
Why "Dismiss" and not "Close"?

Reply:
> Why "Dismiss" and not "Close"?

I wavered back and further between the two. To me you close a window, you dismiss an alert. I could be easily convinced otherwise.

Reply:
> I wavered back and forth between the two. To me you close a window, you dismiss an alert. I could be easily convinced otherwise.

I feel like, given that we made this a separate component, it isn't intended to assume that it is only used within NxAlert.  Additionally we named it NxCloseButton, not NxDismissButton, so I feel like to the same extent that there's a difference in meaning between those two terms, the component name and default label should probably match.

NxAlert can always override it with "Dismiss" if it matters.

Reply:
> > I wavered back and forth between the two. To me you close a window, you dismiss an alert. I could be easily convinced otherwise.
> 
> I feel like, given that we made this a separate component, it isn't intended to assume that it is only used within NxAlert. Additionally we named it NxCloseButton, not NxDismissButton, so I feel like to the same extent that there's a difference in meaning between those two terms, the component name and default label should probably match.
> 
> NxAlert can always override it with "Dismiss" if it matters.

That's a good point about the text matching the component name: https://github.com/sonatype/sonatype-react-shared-components/pull/209/commits/33d171f9818d59eb7f09f13017df4276eecd099e